examples/react-design-system/src/pages/HomepageFullHeadless.jsx

```diff
@@ -480,7 +480,7 @@ const Div24 = styled.div`
   line-height: normal;
   width: calc(33.333333333333336% - 13.333333333333334px);
   margin-left: 20px;
-  @media (max-width: 999px) {
+  @media (max-width: 991px) {
```

Comment:
@mrkoreye, I'll leave this change up to your discretion. It seemed, based on the context, that this example was anticipating the large breakpoint to be 999px so I made the assumption to change it to 991px.